### ISSUE TITLE
cannot expand directories

### ISSUE THREAD

**Issue by Kulak** [scrooloose/nerdtree#566]
I see directory tree being printed with Japanese style characters.

The issue is listed on Stackoverflow:

http://superuser.com/questions/387777/what-could-cause-strange-characters-in-vim/1057425#1057425

My solution was a code change:

```
~/.vim/plugged/nerdtree/plugin]$ git diff NERD_tree.vim
diff --git a/plugin/NERD_tree.vim b/plugin/NERD_tree.vim
index bbcc55f..6342b6c 100644
--- a/plugin/NERD_tree.vim
+++ b/plugin/NERD_tree.vim
@@ -66,7 +66,7 @@ call s:initVariable("g:NERDTreeShowHidden", 0)
 call s:initVariable("g:NERDTreeShowLineNumbers", 0)
 call s:initVariable("g:NERDTreeSortDirs", 1)

-if !nerdtree#runningWindows()
+if nerdtree#runningWindows()
     call s:initVariable("g:NERDTreeDirArrowExpandable", "<E2><96><B8>")
     call s:initVariable("g:NERDTreeDirArrowCollapsible", "<E2><96><BE>")
 else
```

Basically, I have removed character ! from line:

`if !nerdtree#runningWindows()
`

In my case I am having display issue form "Tera Term" terminal over ssh connection to FreeBSD machine.

**Comment by PhilRunninger** [comment]
Have you tried setting this variables in your vimrc?

```
let g:NERDTreeDirArrowExpandable = '▸'
let g:NERDTreeDirArrowCollapsible = '▾'
```
or
```
let g:NERDTreeDirArrowExpandable = '+'
let g:NERDTreeDirArrowCollapsible = '-'
```

**Comment by Kulak** [comment]
I am sorry.  I no longer use VIM and tree, so I cannot easily reproduce the issue.

**The rest of it is out of topic.**

I have switched to [Visual Studio Code](https://code.visualstudio.com/) as an integrated editor platform for multiple programming languages.  I am in particular using it with Go (backend), JavaScript and TypeScript (frontend), HTML, ReactJS all in one project.  It integrates well with GO and TypeScript.  I am also working on embedded C++ project cross-compiled to ARM platform on Linux and Windows.  Its C++ extension saves me there and gives an Ok level of IntelliSense support.

The reason for the switch has been an immense improvement in productivity.  15 years ago I was a VIM hardcore user and tried it again a couple years ago, but I found that it is impossible to remember lots of keyboard shortcuts when working in many programming areas.  VSCode was a much better choice for me (it uses user interaction pattern introduced in Text Mate Mac OS X editor, replicated in Sublime crossplatform editor).
